mamba, mamba-repoquery: add pages (#9056)
tldr-pages/tldr: pages/common/mamba-repoquery.md

# mamba repoquery

> Efficiently query conda and mamba package repositories and package dependencies.
> More information: <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html#repoquery>.

- Search for all available versions of a particular package:

`mamba repoquery search {{package_name}}`

> - Search for all packages satisfying constraints on this query:

Reviewer: Couldn't we remove this part?
```suggestion
- Search for all packages satisfying specific constraints:
```

`mamba repoquery search {{sphinx<5}}`

- List the dependencies of a package installed in the currently activated environment, in a tree format:

`mamba repoquery depends --tree {{scipy}}`

> - Look for other packages who need a particular package installed in the currently activated environment (i.e., inverse of `depends`):

Reviewer 2: ```suggestion
- Print packages in the current environment the require a particular package to be installed (i.e. inverse of `depends`):
```
